docs/docs/contributing/infrastructure.rst

```diff
@@ -0,0 +1,153 @@
+  package from ``PyPI``, deployed via ``gunicorn``.
+  The ansible role to set this up is available at `ansible-role-isso`_. The
+  simple auto-reset feature to curtail vandalism is documented at
+  `isso-demo-config`_.
+
+.. _@posativ: https://github.com/posativ
+.. _posativ.org/isso: https://posativ.org/isso
+.. _isso-comments.de: https://isso-comments.de
+.. _@ix5: https://github.com/ix5
+.. _netcup.de: https://netcup.de
+.. _sphinx-doc: https://www.sphinx-doc.org/
+.. _isso-comments.github.io: https://github.com/isso-comments/isso-comments.github.io
+.. _GitHub Action: https://github.com/isso-comments/isso/blob/master/.github/workflows/build-upload-docs.yml
+.. _comments.isso-comments.de: https://comments.isso-comments.de
+.. _ansible-role-isso: https://git.ix5.org/felix/ansible-role-isso
+.. _isso-demo-config: https://github.com/isso-comments/isso-demo-config
+
+Development
+-----------
+
+Development happens under the `isso-comments GitHub organisation`_, with the
+main `isso-comments/isso`_ repository serving as the source for the Python
+parts, Javascript client, website and documentation, API documentation as well
+as CI/CD configuration.
+
+The ``master`` branch has branch protections set up, requiring an approving
+maintainer review before merging.
+
+.. _isso-comments GitHub organisation: https://github.com/isso-comments
+.. _isso-comments/isso: https://github.com/isso-comments/isso
+
+Packaging
+---------
+
+Isso is released "officially" to as an installable Python package on `PyPI`_
+and as a docker image. Other distributors may package releases of Isso natively
+for operating systems (e.g. for the Arch User repository or formerly Debian),
+but support for these releases should be given by the packager.
+
+.. _PyPI: https://pypi.org/project/isso/
+
+PyPI (Python Package Index)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Release rights for the `isso PyPI`_ project are held by @jelmer, @posativ, @blatinier and @ix5.
+
+Since `PGP support was dropped from PyPI`_ in 2023, PGP keys for signing the
+package are no longer relevant.
+
+.. _isso PyPI: https://pypi.org/project/isso/
+.. _PGP support was dropped from PyPI: https://blog.pypi.org/posts/2023-05-23-removing-pgp/
+
+Docker image
+^^^^^^^^^^^^
+
+The `ghcr.io/isso-comments/isso docker image`_ is rebuilt on every push to
+``master``. Push rights for manually created tags (e.g. ``:release``) are
+inherited from the main ``isso-comments/isso`` GitHub repository (toggle
+``Inherit access from source repository`` turned on).
+
+The `ghcr.io/isso-comments/isso-js-testbed`_ image for running ``Jest``-based
+unit and integration tests is built and pushed manually by @ix5 so far.
+
+.. _ghcr.io/isso-comments/isso docker image: https://github.com/isso-comments/isso/pkgs/container/isso
+.. _ghcr.io/isso-comments/isso-js-testbed: https://github.com/orgs/isso-comments/packages/container/package/isso-js-testbed
+
+Arch User Repository (AUR)
+^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Comment:
This overlaps with our packaging documentation, and isn't part of the upstream project or it's infrastructure.

Reply:
Agreed.

I'll document that in `Prebuilt Packages` in [docs/docs/reference/installation.rst](docs/docs/reference/installation.rst) in a different PR, along with documenting that the Debian package is gone.